mkosi.images/exitrd/mkosi.conf.d/10-debian-ubuntu.conf

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[Match]
+Distribution=|debian
+Distribution=|ubuntu
+
+[Content]
+Packages=
+        systemd
```

Comment:
After the release is out and uploaded to debian I'll add a standalone shutdown package, but it might block all uploads for weeks or months due to NEW so not yet